ivadomed/config_manager.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+
+__ivadomed_dir__ = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Why not `from ivadomed import __ivadomed_dir__`? Redefining `__ivadomed_dir__` could lead to potential problems if for example a file that defines it is moved somewhere else in the package at some point in time (e.g `config_manager` is refactored in `ivadomed/config/config_manager.py`). This variable is not supposed to change so there is not need to redefine it.

I think the most appropriate location to define `__ivadomed_dir__` in the `__init__.py` of the top-level package -> `ivadomed/ivadomed/__init__.py`. This file will never move and always represents the top-level package directory.

Reply:
That's what I did initially (I imported `__ivadomed_dir__` from ivadomed/utils.py), but I had problems with circular import (eb78051). I tried moving `__ivadomed_dir__` to ivadomed/ivadomed/__init__.py, but I currently have the same issue during testing (https://github.com/ivadomed/ivadomed/pull/484/checks?check_run_id=1309819496). @Drulex I am not sure how to fix this, do you have an idea how to fix this? Thank you for your help!

Reply:
Yes will look.

Reply:
Oh I hadn't noticed that `ivadomed/__init__.py` imports `utils.py` because of the git code and then `utils.py` also needs `__ivadomed_dir__`. It's not a big deal, we can just put `__ivadomed_dir__` in `utils.py` and expose it in `ivadomed/__init__.py` the same way as `__version__`.

The problem I see now is that `utils.py` is importing from `config_manager.py`. In fact it is importing from a lot of other files which is a bad idea considering that this is a utilities files (normally code should depend on this, not the other way around).

This is a bigger problem than moving `__ivadomed_dir__` around. I suggest we revert the recent commits and I'll create a separate issue for this.

Reply:
Good thank you for looking into this, I opened an issue here: #489